src/cares_wrap.cc

```diff
@@ -857,18 +832,14 @@ class GetHostByAddrWrap: public QueryWrap {
 };
 
 
-class GetHostByNameWrap: public QueryWrap {
+class GetHostByNameWrap : public QueryWrap {
  public:
   explicit GetHostByNameWrap(Environment* env, Local<Object> req_wrap_obj)
-      : QueryWrap(env, req_wrap_obj) {
-  }
+      : QueryWrap(env, req_wrap_obj) {}
 
   int Send(const char* name, int family) override {
-    ares_gethostbyname(env()->cares_channel(),
-                       name,
-                       family,
-                       Callback,
-                       GetQueryArg());
+    ares_gethostbyname(
+        env()->cares_channel(), name, family, Callback, GetQueryArg());
```

Comment:
I'm not sure how I feel about these types of changes. Personally in my code I will typically put the ending `);` on the next line so you can more easily see where the call ends (similar to braces for conditional blocks).